qutebrowser/config/configtypes.py

```diff
@@ -1996,3 +1996,15 @@ def to_py(
             return urlmatch.UrlPattern(value)
         except urlmatch.ParseError as e:
             raise configexc.ValidationError(value, str(e))
+
+
+class StatusbarWidget(String):
+
+    """A Widget for the status bar.
+
+    Allows some predefined widgets and custom text-widgets via text:$CONTENT."""
```

Comment:
nitpick:

```suggestion
    Allows some predefined widgets and custom text-widgets via text:$CONTENT.
    """
```

(as per [PEP 257](https://www.python.org/dev/peps/pep-0257/))